css/settings-section.scss

```diff
@@ -21,6 +21,11 @@
  *
  */
 
+
+@import 'settings/settings-addressbook';
```

Comment:
I would prefer all imports on the same file :)

Reply:
copy also to the copyright header I will do this when in today as the first thing I do. Sorry forgot about the debouncer I need to think how I transfer these comments to a todo for myself, I loose them on github. I will work through the other points also. The only one is addnewaddressbook that is not my work and I think @suntala has probably updated this on her branch.

Reply:
Arf, then forget about the new addressbook :) :tada: 
Sorry :see_no_evil: